NDTensors/test/NDTensorsTestUtils/device_list.jl

```diff
@@ -16,14 +16,18 @@ if "metal" in ARGS || "all" in ARGS
   Pkg.add("Metal")
   using Metal
 end
+if "cutensor" in ARGS || "all" in ARGS
+  Pkg.add("cuTENSOR")
```

Comment:
I see some funny things in the CI installation process with `TensorOperations` getting downgraded to some random old version of that package. Maybe we should also add `Pkg.rm("TensorOperations")` here. Or to be safe:
```julia
if in("TensorOperations", map(v -> v.name, values(Pkg.dependencies())))
  Pkg.rm("TensorOperations")
end
```
to only remove it if it is installed (see https://discourse.julialang.org/t/new-pkg-how-to-check-if-a-package-is-installed/13141).

Reply:
Yes `TensorOperations` is downgraded which causes an issue but should be fixed when they update their code. I will try removing it and see if that introduces any issues

Reply:
This works I just moved `using TensorOperations` into an if statement that is unreachable if one is testing `cuTENSOR`. Thanks!